### ISSUE TITLE
TDataMember::SetterMethod()

### ISSUE THREAD

**Issue by britzger** [root-project/root#6535]
The analysis software of the H1 Experiment at DESY (you may know it from the tutorial 'h1analysis.C') makes use of `TClass` `RTTI` for parameter setting. With newer releases (probably since root 5.xx?) we get a `NULL` pointer when calling `TDataMember::SetterMethod()`, whereas `TDataMember::GetterMethod()` works well. 
When looking at the source code, there is a fall-back option to guess the `Method-Name  ("SetXYZ()")` and then get it:
```
          if (GetClass()->GetMethod(settername, "1"))
             fValueSetter = new TMethodCall(cl, settername, "1");
```
This does not provide the SetterMethod either. However, it would work well with `"0"` instead of `"1"`. 
The GetterMethod in TDataMember::GetterMethod() is well obtained with "".
No idea if this is a bug, or a feature, and if the "0" option should be included as yet another fallback option?!

**Comment by dpiparo** [comment]
I apologise to revive only now this item. I was curious to understand why this is a need and if with the newer ROOT 6 releases an equally usable solution can be found.

**Comment by britzger** [comment]
Dear Danilo,
after some years I cannot remember the exact details of that ticket any longer. I think, it was related to the "1", "0", or "" option in the constructor of the base method, which probably has changed over time.
Since this issue did not caused any problems any longer in the last years, I believe you can close the ticket.
Thank you for your efforts to improve ROOT!
Best wishes.

**Comment by pcanal** [comment]
This was fixed by b0e71308e080ade1503891354664c288ae0eba31 in v6.30/00 and started as an issue when switching from `CINT` to `Cling`.
